docs/rootless.md

```diff
@@ -23,29 +23,40 @@ penguin:231072:65536
 
 ### Distribution-specific hint
 
+Using Ubuntu kernel is recommended.
+
 #### Debian (excluding Ubuntu)
```

Comment:
I see this was already there, but see my other note about RHEL/CentOS

I'm also not sure if readers would directly consider "Ubuntu" and "Debian" as the same, so it could be an idea to add a separate Ubuntu heading (which could just contain "No preparation steps are needed on Ubuntu". 

Are all versions of Ubuntu supported, or does it only work on (e.g.) 18.04 and up? If so, that might be something to mention.

Reply:
Seems it works on Ubuntu 16.04